test/integration/puppetlabs/services/certificate_authority/ca_true_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns puppetlabs.services.certificate-authority.ca-true-test
+  (:require [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.http.client.sync :as http-client]
+            [clojure.pprint :as pprint]
+  )
+)
```

Comment:
style nitpick - in clojure, all the closing parens, brackets, etc. go on one line, so line 6 would be

```clojure
[clojure.pprint :as pprint]))
```